Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -259,10 +259,6 @@ file that was distributed with this source code.
 
         {% if sonata_admin is defined and sonata_admin_enabled %}
             {% set div_class = div_class ~ ' sonata-ba-field-' ~ sonata_admin.edit ~ '-' ~ sonata_admin.inline %}
-
-            {% if sonata_admin.edit == 'inline' and sonata_admin.inline == 'table' %}
```

Comment:
Well, if it's just a CSS fix, why removing those lines? This is that I'm trying to understand. :)

Yes for the sandbox demo, thanks. :+1: